roles/users/tasks/main.yml

```diff
@@ -65,6 +66,7 @@
     attr_photo: "{{ attr_photo | default(omit) }}"
     state: present
   when: enable_ldap_auth |bool
+  no_log: "{{ hide_password_in_log | default (true) }}"
```

Comment:
`hide_password_in_log` is in the roles default. So no use to provide another default here.